app/Http/routes.php

```diff
@@ -80,6 +80,7 @@
     Route::post("ta/section/new", ["as" => "tasectionnew", "uses" => "TAController@post_section_new", "middleware" => "auth.ta"]);
     Route::post("ta/section/import", ["as" => "tasectionimport", "uses" => "TAController@post_section_import", "middleware" => "auth.ta"]);
     Route::post("ta/section/edit", ["as" => "tasectionedit", "uses" => "TAController@post_section_edit", "middleware" => "auth.ta"]);
+    Route::post("ta/section/bulkcheckin", ["as" => "tasectionbulkcheckin", "uses" => "TAController@post_bulk_checkin_users", "middleware" => "auth.ta"]);
```

Comment:
This should be `"middleware"=> "auth.tutor"` since tutors currently have access to `Sections` in the `TA Console` (ref: https://github.com/Cal-CS-61A-Staff/Check-In/blob/4cd90947364ecf480af61010873b62ca8c3913db/resources/views/ta/console.blade.php#L43). If you really want to keep this as only accessible to TAs you should hide the UI button to trigger the modal.